c7n/resources/networkmanager.py

```diff
@@ -0,0 +1,109 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n.actions.core import BaseAction
+from c7n.manager import resources as c7n_resources
+from c7n.query import QueryResourceManager, TypeInfo, DescribeSource, ConfigSource
+from c7n.utils import local_session, type_schema
+from c7n.tags import RemoveTag, Tag
+
+
+class GetCoreNetwork(DescribeSource):
+
+    def augment(self, resources):
+        resources = super().augment(resources)
+        return resources
+
+
+class DescribeGlobalNetwork(DescribeSource):
+
+    def augment(self, resources):
+        resources = super(DescribeGlobalNetwork, self).augment(resources)
+        return resources
+
+
+@c7n_resources.register('networkmanager-core-network')
+class CoreNetwork(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'networkmanager'
+        enum_spec = ('list_core_networks', 'CoreNetworks', None)
+        detail_spec = (
+            'get_core_network', 'CoreNetworkId',
+            'CoreNetworkId', None)
+        arn = 'CoreNetworkArn'
+        name = 'CoreNetworkId'
+        id = 'CoreNetworkId'
+        date = 'CreatedAt'
+        config_type = None
+        cfn_type = 'AWS::NetworkManager::CoreNetwork'
+        permissions_augment = ("networkmanager:ListTagsForResource",)
+        universal_taggable = object()
+
+    source_mapping = {'describe': GetCoreNetwork, 'config': ConfigSource}
+
+
+@c7n_resources.register('networkmanager-global-network')
+class GlobalNetwork(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'networkmanager'
+        enum_spec = ('describe_global_networks', 'GlobalNetworks', None)
+        arn = 'GlobalNetworkArn'
+        name = 'GlobalNetworkId'
+        id = 'GlobalNetworkId'
+        date = 'CreatedAt'
+        config_type = cfn_type = 'AWS::NetworkManager::GlobalNetwork'
+        permissions_augment = ("networkmanager:ListTagsForResource",)
+        universal_taggable = object()
```

Comment:
No need for this line since 'tag' and 'untag' are implemented.

Reply:
Removed. Thanks!